presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/RewriteSpatialPartitioningAggregation.java

```diff
@@ -71,17 +71,17 @@ public RewriteSpatialPartitioningAggregation(Metadata metadata)
         this.metadata = requireNonNull(metadata, "metadata is null");
     }
 
-    private static boolean hasSpatialPartitioningAggregation(AggregationNode aggregation)
+    private boolean hasSpatialPartitioningAggregation(AggregationNode aggregation)
     {
         return aggregation.getAggregations().values().stream()
-                .map(Aggregation::getCall)
-                .anyMatch(call -> call.getName().toString().equals(NAME) && call.getArguments().size() == 1);
+                .anyMatch(a -> metadata.getFunctionManager().getFunctionMetadata(a.getFunctionHandle()).getName().equals(NAME)
+                        && a.getArguments().size() == 1);
```

Comment:
Put this to the previous line

Reply:
It will be a little long to fit in one line.